### ISSUE TITLE
For loop improvements

### ISSUE THREAD

**Issue by mflorence99** [Alex-Rafter/typeCurve#4]
You could write your `for` loop better as:

```js
for (let i = 0; i < letterForms.length; i++) {
    letterForms[i].style.fontSize = x + "px";
}
```

But it is worth looking at the [array functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array), in particular `forEach` and if you feel particularly adventurous, `map`, `reduce` etc. Using them is much preferred to `for` loops in modern programming. Once you get the hang of the syntax of [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), you can see how incredibly useful and compact they are.

```js
letterForms.forEach(item => item.style.fontSize = x + "px");
```

There's one complication. `querySelectorAll` is 'special' in that it doesn't return an array. So in this case:

```js
Array.from(letterForms).forEach(item => item.style.fontSize = x + "px");
```

The [Mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference) docs are fabulous references and worth bookmarking - even casual reading! And not just for JavaScript either, as they cover all the Web technologies, HTML, CSS etc.

**Comment by Alex-Rafter** [comment]
Because of the later suggestion to optimise by targeting font size changes on the parent grid the for loop changes didn't make it in to branch 1. That being said i had a play around with the new syntax and really like have succint it is compare to the for loop. The Array.from() is very simple and useful too. 

I did have issues with arrow syntax for the callback though. Not sure excatly what i did wrong here. I ended up getting it working but used older anonymous function sytnax. Will have to play around more with this.
